packages/components/src/timeline/util.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+
+const orderByOptions = {
+	ASC: 'asc',
+	DESC: 'desc',
+};
+
+const groupByOptions = {
+	DAY: 'day',
+	WEEK: 'week',
+	MONTH: 'month',
+};
+
+const sortAscending = ( groupA, groupB ) => groupA.datetime - groupB.datetime;
+const sortDescending = ( groupA, groupB ) => groupB.datetime - groupA.datetime;
+
+const sortByDateUsing = ( orderBy ) => {
```

Comment:
Thank you @reykjalin. I like that you're being more explicit here.  This reads nicely.